tubes/lua.lua

```diff
@@ -603,7 +606,7 @@ local function save_memory(pos, meta, mem)
 		meta:set_string("lc_memory", memstring)
 		meta:mark_as_private("lc_memory")
 	else
-		print("Error: lua_tube memory overflow. "..memsize_max.." bytes available, "
+		minetest.log("error", "lua_tube memory overflow. "..memsize_max.." bytes available, "
```

Comment:
```suggestion
		minetest.log("info", "lua_tube memory overflow. "..memsize_max.." bytes available, "
```
This should probably be `info` rather than error, it is part of mod mechanisms instead of real error.

Reply:
what if.... it was put as an error message so you *knew* it was memory, and not some overheating error

Reply:
I mean error stream is meant for actual mod or engine errors and unexpected situations instead of logging normal expected and fully handled situations.
For one `--quiet` command line switch silences info stream but keeps printing to error stream.
Like having "error" written on message is fine if you'd want to distinguish certain game events but actual error stream should be left for actual errors.

Reply:
> I mean error stream is meant for actual mod or engine errors and unexpected situations instead of logging normal expected and fully handled situations. For one `--quiet` command line switch silences info stream but keeps printing to error stream. Like having "error" written on message is fine if you'd want to distinguish certain game events but actual error stream should be left for actual errors.

i meant error message in the luacontroller, sorry for confusion